internal/indexmanagement/reconcile_test.go

```diff
@@ -79,51 +77,6 @@ var _ = Describe("Index Management", func() {
 			})
 		})
 	})
-	Describe("#reconcileCronJob", func() {
```

Comment:
why'd you delete this?

Reply:
`reconcileCronjob` was testing only that the api client and namely the fake client can persist the Cronjob object. Not much value in this.